build.gradle

```diff
@@ -11,7 +11,7 @@ java {
 }
 
 group = 'cloud.eppo'
-version = '4.0.1-SNAPSHOT'
+version = '5.0.0'
```

Comment:
major bump as the static method `stopPolling` has been removed.

Reply:
👍  Makes sense!